src/controllers/controller.doughnut.js

```diff
@@ -305,7 +305,12 @@ module.exports = DatasetController.extend({
 
 		for (i = 0, ilen = arcs.length; i < ilen; ++i) {
 			arc = arcs[i];
-			options = controller ? controller._resolveElementOptions(arc, i) : arc._options;
+			if (controller) {
+				controller._configure();
```

Comment:
I don't think it's need to configure this controller here.
Doughnut chart could have multiple datasets that are all shown in the same chart. Now this `controller` here is for the outmost dataset. It should have been configured already.
Or did I miss something?

Reply:
This method `getMaxBorderWidth` is called by `update` and iterates over every controller. What happens is the first controller to be updated calls `controller._resolveElementOptions` for every controller which fails on all subsequent controllers because they have not yet been configured since they haven't been updated yet

Reply:
👀 ok